src/guiding.jl

```diff
@@ -192,22 +192,22 @@ function _forwardguiding(k::SDEKernel, message, (x0, ll0), alg::Union{Stochastic
 
   if numtraj==nothing
     if !isadaptive
-      sol = solve(prob, alg, tstops=message.ts; kwargs...)
+      sol = solve(prob, alg, tstops=message.ts, adaptive=isadaptive; kwargs...)
     else
       sol = solve(prob, alg, dt=dt, adaptive=isadaptive; kwargs...)
     end
+    return sol.t, [x[1:length(x0)] for x in sol.u], sol.u[end], sol.W, sol[end][end]
```

Comment:
the list comprehension is bad when `x0` is a `StaticArray`. @mschauer What's the best way to fix that? I think it would also be good to return `sol.W.W` to match the internal solver (also here we would probably like to have a slice because then also the noise dimensions will match.)